utils.go

```diff
@@ -14,6 +15,19 @@ func fixPathForm(last string, file string) string {
 		return file
 	}
 
+	if strings.HasPrefix(last, "~") {
+		path := fixDirPath(file)
+		usr, err := user.Current()
+		if err != nil {
+			return path
+		}
+		return strings.Replace(path, usr.HomeDir, "~", 1)
+	}
+
+	if strings.Contains(last, "..") {
```

Comment:
The `../` completion on my machine works fine.
I think that the problem is that you tried to complete after `..` and not after `../`.
If not, can you elaborate which terminal and OS are you using?

Reply:
Oh, we are vendoring a release version for sure as in https://github.com/redhat-developer/odo/blob/master/glide.yaml#L40 where it doesn't seem to work..
But TBH when I last tried to verify it with latest master, it didn't work but now latest master seems to work... may be I did something wrong last time :( 

I'll stash my changes related `..`

Reply:
@posener It doesn't seem to work for me on 

```
[redhat@localhost odo]$ echo $SHELL
/bin/bash

[redhat@localhost odo]$ bash --version 
GNU bash, version 4.4.12(1)-release (x86_64-redhat-linux-gnu)
Copyright (C) 2016 Free Software Foundation, Inc.
License GPLv3+: GNU GPL version 3 or later <http://gnu.org/licenses/gpl.html>

This is free software; you are free to change and redistribute it.
There is NO WARRANTY, to the extent permitted by law.

[redhat@localhost odo]$ cat /etc/*-release
Fedora release 27 (Twenty Seven)
NAME=Fedora
VERSION="27 (Workstation Edition)"
ID=fedora
VERSION_ID=27
PRETTY_NAME="Fedora 27 (Workstation Edition)"
ANSI_COLOR="0;34"
CPE_NAME="cpe:/o:fedoraproject:fedora:27"
HOME_URL="https://fedoraproject.org/"
SUPPORT_URL="https://fedoraproject.org/wiki/Communicating_and_getting_help"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=27
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=27
PRIVACY_POLICY_URL="https://fedoraproject.org/wiki/Legal:PrivacyPolicy"
VARIANT="Workstation Edition"
VARIANT_ID=workstation
Fedora release 27 (Twenty Seven)
Fedora release 27 (Twenty Seven)
```

Reply:
Not even after `../`